src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {},
```

Comment:
- Hey, you've not added your reducers, please, add them. it should look something like this 
`reducer: { books: booksReducer, categories: categoriesReducer }`

Reply:
Thanks so much. oops! Omitted them. Just made the required changes.